frontend/src/app/main/data/workspace/transforms.cljs

```diff
@@ -260,6 +231,39 @@
 
     [root transformed-root ignore-geometry?]))
 
+(defn- set-modifiers-recursive
+  [modif-tree objects shape modifiers root transformed-root ignore-constraints]
+  (let [children (map (d/getf objects) (:shapes shape))
+
+        transformed-shape (gsh/transform-shape (assoc shape :modifiers modifiers))
+
+        [root transformed-root ignore-geometry?]
+        (check-delta shape root transformed-shape transformed-root objects)
+
+        modifiers (assoc modifiers :ignore-geometry? ignore-geometry?)
+
+        transformed-rect (gsh/calc-transformed-parent-rect shape modifiers)
+
+        set-child
+        (fn [modif-tree child]
+          (let [child-modifiers
+                (gsh/calc-child-modifiers shape child modifiers ignore-constraints transformed-rect)]
+
+            (cond-> modif-tree
+              (d/not-empty? (d/without-keys child-modifiers [:ignore-geometry?]))
```

Comment:
the same as previous comment, don't use without-keys if you only have  single key, `dissoc` is much better and more efficient alternative.

```
Execution time sample mean : 31.313933 ns (for dissoc with single argument)
Execution time sample mean : 110.829398 ns (for without-keys with a vector of single element)
```
without-keys is worth only when we have a "large" or dynamic list (vector)

Reply:
ok